File: cwt-spring/src/main/java/com/cwtsite/cwt/domain/game/view/RatingDto.java
package com.cwtsite.cwt.domain.game.view;

import com.cwtsite.cwt.domain.game.entity.Rating;
import com.cwtsite.cwt.domain.game.entity.enumeration.RatingType;

public class RatingDto {

    private Long user;
    private RatingType type;

    public Long getUser() {
        return user;
    }

    public void setUser(Long user) {
        this.user = user;
    }

    public RatingType getType() {
        return type;
    }

    public void setType(RatingType type) {
        this.type = type;
    }

    public static RatingDto toDto(Rating rating) {
        final RatingDto dto = new RatingDto();

        dto.setUser(rating.getUser().getId());
        dto.setType(rating.getType());

        return dto;
    }
}

File: cwt-spring/src/main/java/com/cwtsite/cwt/domain/game/view/RatingDto.kt
package com.cwtsite.cwt.domain.game.view

import com.cwtsite.cwt.domain.game.entity.Rating
import com.cwtsite.cwt.domain.game.entity.enumeration.RatingType
import com.cwtsite.cwt.domain.user.view.model.UserMinimalDto

data class RatingDto(
    val id: Long,
    val user: UserMinimalDto,
    val type: RatingType
) {

    companion object {
        fun toDto(rating: Rating) = RatingDto(
            id = rating.id!!,
            user = UserMinimalDto.toDto(rating.user),
            type = rating.type
        )
    }
}
